.github/workflows/pr_badges.yaml

```diff
@@ -0,0 +1,57 @@
+name: PR badges
+
+on:
+  pull_request:
```

Comment:
We can try to use pull_request_target event here instead of pull_request to make these PR badges work for forked pull requests, see more information on pull_request_target event here: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target. These events run with read/write permission which is needed for the bot to add comments.